### ISSUE TITLE
Add a difficulty called "Clown demon" for levels that use CBF blockers

### ISSUE THREAD

**Issue by ImmediateMurderProfessionals** [ItzKiba/GrandpaDemon#16]
Click between frames is legitimate, so thus, all levels that prevent players from using the mod (such as Nullscapes) should be instantly rated "Clown demon"

**Comment by heavyutils** [comment]
YES YES YES I LOVE UR SUGGESTION (but you do know this mod creator is the same one that made Nullscapes too, right?)

**Comment by ImmediateMurderProfessionals** [comment]
> YES YES YES I LOVE UR SUGGESTION (but you do know this mod creator is the same one that made Nullscapes too, right?)

I am fully aware I’m just teasing him

**Comment by ilovesoggycate** [comment]
no

**Comment by heavyutils** [comment]
> no

yes

**Comment by ilovesoggycate** [comment]
no
